src/components/Header/Header.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Header from './Header';
+import EditorLogo from '../../img/editorLogo.png';
+import TopicsLogo from '../../img/topicsLogo.png';
+
+export default {
+  title: 'Components/Header',
+  component: Header,
+} as ComponentMeta<typeof Header>;
+
+const Template: ComponentStory<typeof Header> = (args) => <Header {...args} />;
+
+export const Editor = Template.bind({});
+Editor.args = {
+  logo: EditorLogo,
+  disclaimerStart: true,
+  appBarColor: 'default',
+  logoWidth: '200',
+  logoHeight: '58',
+  mobileLogoWidth: '130',
+  mobileLogoHeight: '40',
+  withLoginAndSignup: true,
+  diclaimerClick: () => console.log('Disclaimer Click'),
+};
```

Comment:
This is the old logo and can be removed, the header and logo should be the same in all products. Just keep the Topics story as the default (renamed to "Default").

Reply:
Fixed